submission/loglist_refresher_test.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package submission
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/certificate-transparency-go/loglist"
+	"github.com/google/go-cmp/cmp"
+)
+
+func createDataFile(t *testing.T, name string, data []byte) *os.File {
+	t.Helper()
+	f, err := ioutil.TempFile("", name)
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+	if _, err := f.Write(data); err != nil {
+		t.Fatalf("Unable to write into testing file: %v", err)
+	}
+	if err := f.Close(); err != nil {
+		t.Fatalf("Unable to close testing file: %v", err)
+	}
+	return f
+}
+
+func rewriteDataFile(t *testing.T, name string, data []byte) *os.File {
```

Comment:
Could replace this func with `ioutil.WriteFile(name, data, 0755)`.

Reply:
Thanks, indeed.